### PR TITLE
Refactor cli to use entrypoint

### LINKED ISSUE #82: Switch cli to use entry point instead of env vars
Use Docker entry point and refactor cli

### PR DESCRIPTION
**Pull request by m-lukas**
# Description

Refactors cli to use mainly command arguments instead of env variables while leaving `KAFKA_ADDRESS`, `KAFKA_INSTAGRAM_TOPIC` and `KAFKA_TWITTER_TOPIC` for configuration purposes.  Implements the support for command arguments in the docker image.

Fixes #82 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Cleanup (changes in structure but not functionality)

### CODE REVIEW

**Review by Urhengulas — CHANGES_REQUESTED** [review]
Are you going to add a usage example to docker-compose?

**Review by Urhengulas — CHANGES_REQUESTED** [review]
And it might be useful to add some kind of configurable sleep to be able to directly insert a username at `make run`.

File: cli/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,5 @@ FROM alpine
 RUN apk --no-cache add ca-certificates
 WORKDIR /app
 COPY --from=builder /app/smag-cli .
-CMD ["./smag-cli"]
\ No newline at end of file
+ENTRYPOINT ["./smag-cli"]
+CMD [ "" ]
```
Comment (alexmorten) [inline cli/Dockerfile]:
```suggestion
```
you don't need the `CMD`

**Comment by m-lukas** [comment]
> Are you going to add a usage example to docker-compose?

No because some people might not know that it's executed via docker-compose in the future and therefore it's better to let everyone execute it on its own. However, I added an explanation how to add it to the README to give a hint how to add it.

**Comment by m-lukas** [comment]
I would just leave the CMD in because while it seems that `alpine` has no default command, it just erases the command if there is one and is already tested.

File: cli/Dockerfile (hunk 1 above)
Comment (m-lukas) [inline cli/Dockerfile]:
```suggestion
CMD [ "" ] # optional explicit statement
```

**Review by Urhengulas — APPROVED** [review]

**Pull request merged**